venues/ICLR.cc/2019/Conference/python/invitations.py

```diff
@@ -236,6 +236,51 @@
     public_comment_template['process'] = f.read()
 
 
+public_comment_template = {
```

Comment:
it this part of the bids?

Reply:
there is already a public comment template, right above this